openedx/core/djangoapps/organization_api/urls.py

```diff
@@ -0,0 +1,19 @@
+"""
+URLs for the organization app.
+"""
+from django.conf.urls import patterns, url
+
+from openedx.core.djangoapps.organization_api.api import views
+
+
+ORGANIZATION_KEY_PATTERN = r'(?P<organization_key>[^/]*)'
```

Comment:
@zubair-arbi organization key should not have spaces. It should be a string with only alphanumerics.